examples/cluster-configuration/services-configuration.yaml

```diff
@@ -22,6 +22,9 @@
 #    cluster-id: pai
 #    cluster-type: yarn
 #
+#    # Gateway address, used to visit the cluster from public network
+#    gateway-uri: "https://pai.domain.com"
```

Comment:
Do we need to point the scheme names here. Or use can just config the gateway address. Then both http and https are supported . If gateway supports multi protocol.